kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -551,7 +513,8 @@
             'createSnackbar',
             this.sectionDeletedNotification$({ section_title })
           );
-          this.focusActiveSectionTab();
+          // Run after the focus change of KModal destroyed method
+          setTimeout(() => this.focusActiveSectionTab());
```

Comment:
Is this doing the same kind of thing you'd do with [`Vue.$nextTick`](https://v2.vuejs.org/v2/api/#Vue-nextTick)?

Reply:
I tried even with 4+ consecutive $nextTick calls and all of them were beign called before the setTimeout that excecutes [this focus on KModal destroyed mehod](https://github.com/AlexVelezLl/kolibri-design-system/blob/5d0a1cb83527bb554389fb45cb3e835ae70d118c/lib/KModal.vue#L281). Since there we use setTimeout, we need to use setTimeout also here, for the focusActiveSectionTab method to be enqueued after the lastFocus.focus() in KModal.

Reply:
Ah right on! Thanks for the explanation.